### ISSUE TITLE
log4j information in generated ttl

### ISSUE THREAD

**Issue by smoretti** [sib-swiss/sparql-examples#43]
Hi

I run `java -jar sparql-examples-utils.jar convert -i examples/ -p MetaNetX -f ttl >examples.ttl` and found log4j information in the generated ttl file.
I guess those lines should not be there

E.g.
```
log4j: reset attribute= "false".
log4j: Threshold ="null".
log4j: Retreiving an instance of org.apache.log4j.Logger.
log4j: Setting [swiss.sib] additivity to [true].
log4j: Level value for swiss.sib is  [debug].
log4j: swiss.sib level set to DEBUG
log4j: Retreiving an instance of org.apache.log4j.Logger.
log4j: Setting [org.eclipse.rdf4j] additivity to [true].
log4j: Level value for org.eclipse.rdf4j is  [warn].
log4j: org.eclipse.rdf4j level set to WARN
log4j: Retreiving an instance of org.apache.log4j.Logger.
log4j: Setting [org.apache.jena] additivity to [true].
log4j: Level value for org.apache.jena is  [warn].
log4j: org.apache.jena level set to WARN
log4j: Retreiving an instance of org.apache.log4j.Logger.
log4j: Setting [com.bigdata] additivity to [true].
log4j: Level value for com.bigdata is  [warn].
log4j: com.bigdata level set to WARN
log4j: Level value for root is  [debug].
log4j: root level set to DEBUG
log4j: Class name: [org.apache.log4j.ConsoleAppender]
log4j: Parsing layout of class: "org.apache.log4j.PatternLayout"
log4j: Setting property [conversionPattern] to [%t %d{yy-MM-dd HH:mm:ss Z} %c%n  %p: %m%n].
log4j: Adding appender named [console] to category [root].

_:genid-0bc31781c121477892806179333debd62-sparql_examples_prefixes a <http://www.w3.org/2002/07/owl#Ontology>;
  <http://www.w3.org/2000/01/rdf-schema#comment> "This is a collection of SPARQL queries that are examples for use on different websites.";
  <http://www.w3.org/2002/07/owl#imports> <http://www.w3.org/ns/shacl#>;

```

**Comment by JervenBolleman** [comment]
identical issue to https://github.com/sib-swiss/sparql-examples-utils/issues/17, could you try again with the updated code?

**Comment by smoretti** [comment]
Yes, @vemonet let me know afterwards, sorry.

I have update to version v2.0.10, but still have the same issue when I convert to ttl.

(all tests pass)

**Comment by JervenBolleman** [comment]
The fix is not in a released version 2.0.10, it would appear in 2.0.11

**Comment by smoretti** [comment]
It is now fixed with 2.0.11 !

Thanks
